docs/manage-audit-log-filter.md

```diff
@@ -0,0 +1,22 @@
+# Manage the Audit Log Filter files
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The Audit Log Filter files have the following potential results:
+
+* Consume a large amount of disk space
+* Grow large
+
+You can manage the space by using log file rotation. This operation renames and then rotates the current log file and then uses the original name on a new current log file. You can rotate the file either manually or automatically.
+
+If automatic rotation is enabled, you can prune the log file. This pruning operation can be based on either the log file age or combined log file size.
+
+## Manual log rotation
+
+The default setting for [`audit_log_filter_rotate_on_size`](audit-log-filter-variables.md#audit_log_filter_rotate_on_size) is 1GB. The audit log filter plugin does not do an automatic rotation of the log file. You must do the rotation manually with this setting.
```

Comment:
Please change it to something similar to the following:

"When the audit_log_filter_rotate_on_size is set to 0, the audit log filter plugin does not do an automatic rotation of the log file. You must do the rotation manually with this setting."